### PR TITLE
add preset for man_made=crane

### PR DESCRIPTION
**Pull request by willemarcel**

### CODE REVIEW

**Comment by willemarcel** [comment]
Solves https://github.com/openstreetmap/iD/issues/4374

**Comment by bhousel** [comment]
looks good.. can you rename 
`data/presets/fields/crane_type.json`
to 
`data/presets/fields/crane/type.json`

This is the file naming convention when the key is namespaced like `crane:type=*`

File: data/presets/fields.json [hunk 1]
```diff
@@ -472,6 +472,18 @@
             "type": "typeCombo",
             "label": "Type"
         },
+        "crane_type": {
+            "key": "crane:type",
+            "type": "combo",
+            "label": "Crane Type",
+            "strings": {
+                "options": {
+                    "portal_crane": "Portal crane",
```
Comment (bhousel) [inline data/presets/fields.json]: Can you title case:  "Portal Crane"

File: data/presets/fields.json [hunk 2]
```diff
@@ -472,6 +472,18 @@
...
+                    "floor-mounted_crane": "Floor-mounted crane",
```
Comment (bhousel) [inline data/presets/fields.json]: tile case:  Floor-mounted Crane

**Comment by willemarcel** [comment]
@bhousel Done! Thanks for the review!

**Comment by bhousel** [comment]
Thanks @willemarcel, I fixed the path and merged this 👍

**Pull request merged**